dashboard/test/ui/features/share_buttons.feature

```diff
@@ -0,0 +1,23 @@
+@as_student
+  # need @as_student to log in in order to get to Game Lab since it's a droplet game
+
+Feature: Share Buttons
+  # Making How It Works button show up in Sprite Lab and not in Game Lab
+
+Scenario: How It Works Button appears for Sprite Lab share page
+  Given I am on "http://studio.code.org/projects/spritelab"
+  And I wait for the page to fully load
+  When I navigate to the shared version of my project
+  # up till here gets you to that share page with the mobile screen but after that it takes you to the homepage
```

Comment:
I'm not sure I understand what you mean in this comment. What do you mean by 'after that it takes you to the homepage'?

Reply:
I'll go ahead and take out that comment. That comment was written when I  only had line 9 and 10 to test things around to see what each step does. So at the moment, after that line 10 it went to the share page URL then concluded the test by going back to the homepage.